jetson/imu_channel/README.md

```diff
@@ -58,17 +58,49 @@ You should see a stream of `IMUData` structs being printed to the terminal, try
 
-Now just detach the BNO055 from its mount and wave it around in the air. Watch [this video](https://www.youtube.com/watch?v=Bw0WuAyGsnY) on what patterns to move it in for optimal results. You'll know you're done when all three calibration status variables read 3.
-
-_(There's also a 4th calibration status called sys, but that one usually sorts itself out. I'll add it to the LCM struct in the future)_
+Be aware that the accelerometer can be very tricky to calibrate, sometimes taking upwards of 10 minutes to get the calibration status to 3. For fastest calibration, follow the pattern in the video *very* slowly, rotating between angles slowly and holding the IMU still at those angles for around 10 seconds each.  It's also worth noting that the magnetometer calibration status often drops to a lower value (we've seen anywhere from 2 to 0) after running it for a while.
 
 The BNO055 should now be ready to go and outputting good data, so you can run whatever other programs need to use it.
 
+## Arduino Pro CLI
+
+Instead of using the Arduino IDE to upload to the Arduino, you can use the Arduino Pro CLI tool. This is better since it can be done over SSH purely from the command line.
+
+### Installation
+Download the `arduino-cli` binaries from [here](https://arduino.github.io/arduino-cli/0.21/installation/), then put them in `/bin/`
+
+Install the BNO055 library: \
+`arduino-cli lib install "Adafruit BNO055"`
+
+Install `screen` with `sudo apt install screen`
+
+### Compile/Upload
+First install the necessary core for your board: \
+`arduino-cli core install arduino:megaavr` 
+
+Then find your board: \
+`arduino-cli board list`
+
+Then compile, specifying the board type and the sketch path: \
+`arduino-cli compile -b arduino:megaavr:nona4809 Desktop/my_sketch.ino`
+
+Then upload, specifying the port, board type, and sketch: \
+`arduino-cli upload -p /dev/ttyACM0 -b arduino:megaavr:nona4809 \Desktop/my_sketch.ino`
+
+(the Arduino Nano Every uses board type `arduino:megaavr:nona4809`)
+
+### Viewing Serial Output
+Use GNU screen tool, specifying port and baud rate: \
+`screen /dev/ttyACM0 9600`
+
+Exit screen by pressing _ctrl+a_ then _k_
+
+
 ## TODO
-- update `IMUData` LCM struct to include all 4 calibration status values and maybe get rid of values we don't need?
 - add feature to allow axes to be rearranged easily (to account for different sensor orientation relative to rover)
-- try it out on Arduino IDE 2.0 and change tutorial to work with that, or try it on Arduino Pro CMD and write a script that does everything using that
\ No newline at end of file
+- try it out on Arduino IDE 2.0 and change tutorial to work with that
+- write a script that uses arduino cli to do everything automatically
```

Comment:
would be very useful! could be modified to work for the spectral triad too ☠️

Reply:
yeah thats exactly what I was thinking, should be like 2 lines long if arduino CLI and all the necessary libs are already installed on the base station